tests/console/commands/test_init.py

```diff
@@ -144,7 +144,7 @@ def test_interactive_with_git_dependencies(app, repo, mocker, poetry):
     command = app.find("init")
     command._pool = poetry.pool
 
-    mocker.patch("poetry.utils._compat.Path.open")
+    # mocker.patch("poetry.utils._compat.Path.open")
```

Comment:
If you have not considered it yet, you could consider adding a test fixture to do this since this is reused. Something like this might be useful.

```py
@pytest.fixture
def mocked_pyproject_toml(mocker):
    def side_effect(self, *args, **kwargs):
        if self.name == "pyproject.toml":
            return MagicMock()
        return self.open(*args, **kwargs)

    patched_open = mocker.patch("poetry.utils._compat.Path.open")
    patched_open.side_effect = side_effect
    yield patched_open
```

Reply:
This was already added for another change at https://github.com/python-poetry/poetry/blob/95e3490a76c876f2e5487499fa592cd434ddc901/tests/conftest.py#L122-L134

Could reuse this.